elements/pl-pivot-table/pl-pivot-table-double.mustache

```diff
@@ -0,0 +1,106 @@
+{{#question}}
+        <script>
+            var uuid = "{{uuid}}";
+            var num_row_dropzone = parseInt("{{num_row_dropzone_val}}");
+            var num_index = 2;
+            var is_multicol = "True" === "{{is_multicol}}";
+        </script>
+        <div class="container column-box">
+            <h2>Choose answer column here</h2>
+            {{#column_set}}
+            <div class="row col-set" id="{{uuid}}" data-order="{{order_col}}">
+                {{#column}}
+                <div class="col-md-{{{width}}} col-cell">{{{inner_html}}}</div>
+                {{/column}}
+                {{#is_ellipsis}}
+                <div class="col-md-{{{width}}} col-cell">...</div>
+                {{/is_ellipsis}}
+            </div>
+            {{/column_set}}
+        </div>
+
+        <div class="container index-box">
+            <h2>Choose answer index here</h2>
+            <div class="row">
+                {{#indice_set}}
+                <div class="col-md-1"{{{calibration}}}>
+                    <div class="index-set" id="{{uuid}}" data-order="{{order_index}}">
+                        {{#index}}
+                        <div class="row">
+                            <div class="col-md-12 index-cell" style="height: 30px;">{{{inner_html}}}</div>
+                        </div>
+                        {{/index}}
+                        {{#is_ellipsis}}
+                        <div class="row">
+                            <div class="col-md-12 index-cell" style="height: 30px;">...</div>
+                        </div>
+                        {{/is_ellipsis}}
+                    </div>
+                </div>
+                {{/indice_set}}
+            </div>
+
+        </div>
+
+        <div class="container row-box">
+            <h2>Choose answer row here</h2>
+            
+            <div class="row">
+                {{#row_set}}
+                <div class="col-md-{{{width}}}">
+                    <div class="row-set" id="{{uuid}}" data-order="{{order_row}}">
+                        <div class="row">
+                            <div class="col-md-12 row-cell">&nbsp</div>
+                        </div>
+                        {{#row}}
+                        <div class="row">
+                            <div class="col-md-12 row-cell">{{{inner_html}}}</div>
+                        </div>
+                        {{/row}}
+                        {{#is_ellipsis}}
+                        <div class="row">
+                            <div class="col-md-12 row-cell">...</div>
+                        </div>
+                        {{/is_ellipsis}}
+                    </div>
+                </div>
+                {{/row_set}}
+            </div>
+        </div>
+
+        <div class="container drop-zone">
+            <h2>Drop your answer here</h2>
```

Comment:
there is a huge amount of stuff in this template that is fixed and not customizable.  it seems to me that at a minimum, things like the prompts (names of sections such as "Drop your answer here") should be customizable.  many elements allow this; look at `pl-order-blocks` as an example.

Reply:
This is resolved